codeops-scripts/sync-pr-autoflow-to-repos.ps1

```diff
@@ -89,11 +89,25 @@ function _repoChanges($OrgName, $RepoName)
                     tag = "preview";
                     increment = "patch";
                 }
+                "dependabot-pr" = [ordered]@{
+                    regex = "^dependabot";
+                    tag = "dependabot";
+                    "source-branches" = @("develop", "master", "release", "feature", "support", "hotfix");
```

Comment:
Do you think we should have `main` in this list, to ensure we cover the latest gitversion defaults?

Reply:
Initially I did, until I tried it, at which point I remember that this isn't Git branch names, it's the name of GitVersion branch configurations. So far, all of the repos we've moved from `master` to `main` have kept the GitVersion configuration name as `master` because a load of baked-in convention in GitVersion just assumes your main branch configuration is called `master`.

So we're sticking with what appears to be the standard practice of having a GitVersion branch config called `master` which has a regex of `^main$`,  at which point I believe that `source-branches` content is correct.